File: home/wasa/bin/scrape.rb
#!/usr/bin/env ruby
require 'nokogiri'
require 'open-uri'

url = 'https://www.facebook.com/DiefetteKuh'
#doc = Nokogiri::HTML(open(url))
doc = Nokogiri::HTML(open(url) {|f| f.read})

# for some very weird reasons open gives us the interesting content in
# comments which have to be parsed again it seems
comments = []
doc.root.traverse do |elem|
  if elem.content.include? 'BURGER'
    comments << Nokogiri::HTML::DocumentFragment.parse(elem.content)
  end
end

puts comments[0].at_css('div[role=article] .userContent').content
puts comments[0].at_css('div[role=article] .scaledImage img')['src']

File: home/wasa/bin/scrape.py
#!/usr/bin/env python
import lxml.html
import requests

url = 'https://www.facebook.com/DiefetteKuh'
doc = lxml.html.document_fromstring(requests.get(url).content)

# for some very weird reasons open gives us the interesting content in
# comments which have to be parsed again it seems
comments = [lxml.html.document_fromstring(element.text)
            for element in doc.iter()
            if element.text and 'BURGER' in element.text]

print(comments[0].cssselect('div[role=article] .userContent')[0].text_content())
print(comments[0].cssselect('div[role=article] .scaledImage img')[0].get('src'))
